src/main/kotlin/no/nav/syfo/application/VedtakService.kt

```diff
@@ -59,6 +59,13 @@ class VedtakService(
         return createdVedtak
     }
 
+    fun ferdigstillVedtak(
+        vedtak: Vedtak,
+        veilederident: String,
+    ) {
+        vedtakRepository.setVedtakFerdigstilt(vedtak, veilederident)
+    }
+
```

Comment:
Siden dette er business logikk synes det kan være fint å involvere domenet objektet her. Slik at det man persisterer er det endrede domene objektet.
Synes også det er fint at funksjoner returnerer dataen de har lagret/endret/hentet. Synes testing blir enklere, så får vi også muligheten (om vi vil) å bruke resultatet av operasjonen videre.
```suggestion
    fun ferdigstillVedtak(
        vedtak: Vedtak,
        veilederident: String,
    ): Vedtak {
    	val ferdigbehandletVedtak = vedtak.ferdistill()
        vedtakRepository.setVedtakFerdigstilt(ferdigbehandletVedtak, veilederident)
    }

```